feature-libs/organization/user-registration/components/form/user-registration-form.component.html

```diff
@@ -0,0 +1,287 @@
+                  'userRegistrationForm.fields.addressLine.placeholder'
+                    | cxTranslate
+                }}"
+                formControlName="line1"
+              />
+            </label>
+          </div>
+
+          <div class="form-group">
+            <label>
+              <span class="label-content">{{
+                'userRegistrationForm.fields.secondAddressLine.label'
+                  | cxTranslate
+              }}</span>
+              <input
+                type="text"
+                class="form-control"
+                placeholder="{{
+                  'userRegistrationForm.fields.secondAddressLine.placeholder'
+                    | cxTranslate
+                }}"
+                formControlName="line2"
+              />
+            </label>
+          </div>
+
+          <div class="row">
+            <div class="form-group col-md-6">
+              <label>
+                <span class="label-content">{{
+                  'userRegistrationForm.fields.city.label' | cxTranslate
+                }}</span>
+                <input
+                  type="text"
+                  class="form-control"
+                  placeholder="{{
+                    'userRegistrationForm.fields.city.label' | cxTranslate
+                  }}"
+                  formControlName="town"
+                />
+              </label>
+            </div>
+
+            <div class="form-group col-md-6">
+              <label>
+                <span class="label-content">{{
+                  'userRegistrationForm.fields.postalCode.label' | cxTranslate
+                }}</span>
+                <input
+                  type="text"
+                  class="form-control"
+                  placeholder="{{
+                    'userRegistrationForm.fields.postalCode.placeholder'
+                      | cxTranslate
+                  }}"
+                  formControlName="postalCode"
+                />
+                <cx-form-errors
+                  [control]="registerForm.get('postalCode')"
+                ></cx-form-errors>
+              </label>
+            </div>
+
+            <ng-container
+              *ngIf="regions$ | async as regions"
+              formGroupName="region"
+            >
+              <ng-container *ngIf="regions.length !== 0">
+                <div class="form-group col-md-12">
+                  <label>
+                    <span class="label-content">{{
+                      'userRegistrationForm.fields.state.label' | cxTranslate
+                    }}</span>
+                    <ng-select
+                      class="region-select"
+                      formControlName="isocode"
+                      [searchable]="true"
+                      [clearable]="false"
+                      [items]="regions"
+                      bindLabel="{{ regions[0].name ? 'name' : 'isocode' }}"
+                      bindValue="{{ regions[0].name ? 'isocode' : 'region' }}"
```

Comment:
I can see in console `ERROR Error: Cannot find control with name: 'isocode'` because of that. Why not just:
```suggestion
                      bindLabel="name"
                      bindValue="isocode"
```